config/CUSTOM-RULES.md

```diff
@@ -117,6 +117,20 @@ Custom rules can have more complicated behavior. They can also include other rul
 
 `Little Cup` - allow only Pokémon that can evolve and aren't evolved
 
+`Gen 8 STABmons` - allow Pokémon to use almost any move that matches their typing
```

Comment:
I'd probably put these in separate sections.

Also I'd probably just note that any tier can be used by name, instead of listing all the tiers.